bcgov/HOUS-permit-portal
# Feature/bphh 1234/version diff

## Description

Show a clickable changes box to review manager on the permit template when navigating from the "changes on template" notification.

ticket:
https://hous-bssb.atlassian.net/browse/BPHH-1234

## Steps to QA:

See the card. Add and remove requirements as super admin, save, log in as review manager, navigate using hte notificaiton, see the changes in the floating sidebar. Clicking links should navigate to the appropriate block if it is still present.
